src/components/navbar.tsx

```diff
@@ -1,63 +1,82 @@
-  setShowSettings: (show: boolean) => void;
-}
+const VerticalNavbar = () => {
+  const router = useRouter();
+
+  const navItems = [
+    { path: "/room/1", label: "Raum 1" },
+    { path: "/room/2", label: "Raum 2" },
+  ];
+
+  const isActive = (path: string) => {
+    return router.asPath === path;
+  };
 
-const Navbar: FunctionComponent<NavbarProps> = ({ setShowSettings }) => {
   return (
-    <nav className="navbar navbar-expand-lg navbar-dark bg-dark">
-      <div className="container">
-        <Link href="/">
-          <span className="navbar-brand">TUC Forum</span>
-        </Link>
-        <button
-          className="navbar-toggler"
-          type="button"
-          data-bs-toggle="collapse"
-          data-bs-target="#navbarNav"
-          aria-controls="navbarNav"
-          aria-expanded="false"
-          aria-label="Toggle navigation"
+    <Navbar bg="primary" className="d-flex flex-column vh-100">
+      <Navbar.Brand
+        as={Link}
+        href="/"
+        className="mx-auto"
+        style={{ cursor: "pointer" }}
+      >
+        <Image
+          src={logo}
+          alt="Logo"
+          width={50}
+          height={50}
+          className="rounded"
+        />
+      </Navbar.Brand>
+      <Nav className="flex-column d-flex flex-grow-1">
+        {/* nav items */}
+        <Nav.Link as={Link} href="/" active={isActive("/")} className="mx-auto">
+          Home
+        </Nav.Link>
+        <Nav.Link
+          as={Link}
+          href="/room/join"
+          active={isActive("/room/join")}
+          className="mx-auto"
         >
-          <span className="navbar-toggler-icon"></span>
-        </button>
-        <div className="collapse navbar-collapse" id="navbarNav">
-          <ul className="navbar-nav">
-            <li className="nav-item">
-              <Link className="nav-link" href="room/1">
-                Room 1
-              </Link>
-            </li>
-            <li className="nav-item">
-              <Link className="nav-link" href="room/2">
-                Room 2
-              </Link>
-            </li>
-            <li className="nav-item">
-              <Link className="nav-link" href="room/3">
-                Room 3
-              </Link>
-            </li>
-            <li className="nav-item">
-              <Link className="nav-link" href="room/4">
-                Room 4
-              </Link>
-            </li>
-            <li className="nav-item">
-              <Link className="nav-link" href="room/5">
-                Room 5
-              </Link>
-            </li>
-            <li className="nav-item" onClick={() => setShowSettings(true)}>
-              <span className="nav-link">Settings (user icon)</span>
-            </li>
-          </ul>
+          Entdecken
+        </Nav.Link>
+        <hr className="mx-auto w-75" /> {/* Trenner nach Home */}
+        {/* sorgt dafür dass Settings ganz unten ist*/}
+        <div className="flex-grow-1">
```

Comment:
bezüglich der Ersetzung von `<hr/>`:
- hier würde ich mit Bootstrap eine Border und oben einfügen, die dann dadurch Striche an der Stelle von den `<hr/>`s einfügt.
- die erstreckt sich in der aktuellen Version über die gesamte Breite, aber nicht wenn wir Padding (wie im Kommentar oben erwähnt) hinzfügen